crates/lib/src/qpu/quilc/isa/operator.rs

```diff
@@ -18,23 +18,47 @@ pub(crate) fn wildcard(node_id: Option<i32>) -> Operator {
 pub(crate) enum Operator {
     Gate {
         operator: String,
+        #[serde(deserialize_with = "deserialize_duration_null_default")]
```

Comment:
Can you not just do this and leave out the manual deserializer stuff?
```suggestion
        #[serde(default = "PERFECT_DURATION")]
```

Reply:
`default` expects a function, so it's not quite _that_ simple, but it can be improved with e.g.
```rust
fn deserialize_duration_null_default() -> f64 {
    PERFECT_DURATION
}
```

Reply:
I saw you added `simplify default` and `Revert "simplify default"` commits, so I tried using a default function locally. Looks like an explicit `null` is considered invalid for a `f64` field. So it looks like the deserializer is needed after all :/